### PR TITLE
Remove stale code and change ordering method

### PR DESCRIPTION
**Pull request by Caleb-T-Owens**
## Status

* Current Status: Ready for (front-end / tech) review

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?

- Removes dropdowns from primary certificate page (request from Dapo)
- Fixes completed achievements not showing up after changing pathway
- Removes old ordering method that primary certificate used in favor of the same one which Secondary has
- Removes stale code that was supporting having pathway-less primary and secondary

### CODE REVIEW

File: app/views/certificates/primary_certificate/_achievement_list.html.erb [hunk 1]
```diff
@@ -40,14 +40,16 @@
...
+      <strong>Choose at least <%= group.required_for_completion.humanize %> activity</strong> to <%= group.title.downcase %>
     </p>
-    <%= render 'certificates/pathways/community_achievements',
-      programme_activities: split_community_pathway_activities_by_visibility(
-        group: group,
-        pathway_activities: @recommended_community_activities,
-      ),
-      community_achievements: @community_achievements
-    %>
+    <% group.order_programme_activities_for_user(current_user).each do |programme_activity| %>
```
Comment (Caleb-T-Owens) [inline app/views/certificates/primary_certificate/_achievement_list.html.erb:45]: There are a lot of similarities between all of the X/_achievement_list.html.erb files, at some point we probably want to think of a reasonable abstraction for them. For now, I think we're best to stick to duplication

File: spec/factories/programmes/primary_certificate.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
     slug { 'primary-certificate' }
     description { 'This is the Primary programme' }
     enrollable { true }
+    pathways { create_list(:pathway, 3) }
```
Comment (Caleb-T-Owens) [inline spec/factories/programmes/primary_certificate.rb:7]: Primary and secondary should always have pathways for people to enrol onto

File: spec/factories/user_programme_enrolments.rb [hunk 3]
```diff
@@ -3,10 +3,17 @@
     user
     programme
 
+    before(:create) do |enrolment|
+      if enrolment.programme.pathways? && enrolment.pathway.nil?
```
Comment (Caleb-T-Owens) [inline spec/factories/user_programme_enrolments.rb:7]: If a pathway hasn't been explicitly picked when creating an enrolment to primary or secondary, we pick the first pathway form those programmes

**Review by dcorking — APPROVED** [review]
LGTM 🚂

**Pull request merged**